src/python/pants/backend/python/rules/pex_from_targets.py

```diff
@@ -99,6 +99,29 @@ class TwoStepPexFromTargetsRequest:
     pex_from_targets_request: PexFromTargetsRequest
 
 
+def requirements_pex_from_targets_request(
```

Comment:
How about making this a classmethod? Imo, that would be more discoverable, and we generally seem to prefer factory classmethods in this repo.

Reply:
Classmethod on what class?

Reply:
On `PexFromTargetsRequest`. Something like `PexFromTargetsRequest.requirements_pex()`.